ui/raidboss/data/05-shb/raid/e11s.txt

```diff
@@ -89,49 +96,142 @@ hideall "--sync--"
+381.8 "Burnout" sync /:Fatebreaker's Image:567C:/
+385.6 "Blastburn" sync /:Fatebreaker's Image:567A:/
+386.4 "--targetable--"
+
+397.1 "Elemental Break" sync /:Fatebreaker:566[368]:/
+399.6 "Sinsight/Sinsmite" sync /:Fatebreaker:566[579]:/
 409.6 "Burnished Glory" sync /:Fatebreaker:56A4:/
 
 427.1 "Turn Of The Heavens" sync /:Fatebreaker:566[AB]:/ window 50,50
-429.6 "--middle--" sync /:Fatebreaker:5908:/
-438.6 "Elemental Break" sync /:Fatebreaker:566[36]:/
-441.1 "Sinsmoke/Sinsmite" sync /:Fatebreaker:566[57]:/
+429.6 "--center--" sync /:Fatebreaker:5908:/
+438.6 "Elemental Break" sync /:Fatebreaker:566[368]:/
+441.1 "Sinsmite/Sinsmoke" sync /:Fatebreaker:56(5A|57|65|67):/
 450.6 "Powder Mark" sync /:Fatebreaker:56A2:/
 458.1 "Right Of The Heavens" sync /:Fatebreaker:566[EF]:/
-460.6 "--middle--" sync /:Fatebreaker:5908:/
+460.6 "--center--" sync /:Fatebreaker:5908:/
 469.5 "Burn Mark" sync /:Fatebreaker:56A3:/
-469.5 "Bound Of Faith" sync /:Fatebreaker:565F:/
-471.6 "Solemn Charge" sync /:Fatebreaker:5660:/
-472.9 "Sinsight" sync /:Fatebreaker:5661:/
-478.5 "Mortal Burn Mark" sync /:Fatebreaker:5662:/
-488.3 "--middle--" sync /:Fatebreaker:5908:/
-
-493.7 "Prismatic Deception" sync /:Fatebreaker:5672:/
+469.5 "Bound Of Faith" sync /:Fatebreaker:565[8F]:/
+471.6 "Solemn Charge" sync /:Fatebreaker:56(59|60):/
+472.9 "Sinsight/Sinsmoke" sync /:Fatebreaker:56[56][159A]:/
+478.5 "Mortal Burn Mark?" # sync /:Fatebreaker:5662:/
+488.3 "--center--" sync /:Fatebreaker:5908:/
+
+493.7 "Prismatic Deception" sync /:Fatebreaker:5672:/ window 493.7,10
+496.8 "--untargetable--"
+513.9 "Blasting Zone" sync /:Fatebreaker's Image:56A5:/
+532.0 "Blasting Zone" sync /:Fatebreaker's Image:56A5:/
+537.9 "--targetable--"
 549.2 "Burnished Glory" sync /:Fatebreaker:56A4:/ window 30,30
 
-561.7 "--middle--" sync /:Fatebreaker:5908:/
-570.7 "Cycle of Faith 1"
-# TODO: three groups of mechanics
+
+561.7 "--center--" sync /:Fatebreaker:5908:/
+570.7 "Cycle Of Faith" sync /:Fatebreaker:5692:/ jump 700
+570.7 "--sync--" sync /:Fatebreaker:569A:/ jump 800
+570.7 "--sync--" sync /:Fatebreaker:568B:/ jump 900
+570.8 "Elemental Break"
+573.3 "Sinsight/Sinsmite/Sinsmoke"
+575.2 "Burnt Strike?"
+575.5 "Blastburn?"
+580.2 "Shining Blade?"
+580.2 "Floating Fetters"
+
+
+
+# Lightning Cycle
+691.0 "--center--" sync /:Fatebreaker:5908:/
+700.0 "Cycle Of Faith" sync /:Fatebreaker:5692:/
+700.1 "Elemental Break" sync /:Fatebreaker:5693:/
+702.7 "Sinsmite" sync /:Fatebreaker:5694:/
+704.5 "Burnt Strike" sync /:Fatebreaker:5695:/
+706.2 "Burnout" sync /:Fatebreaker:5696:/
+709.6 "Floating Fetters" sync /:Fatebreaker:5920:/
+711.6 "Solemn Charge" sync /:Fatebreaker:5697:/
+713.1 "Sinsmite" sync /:Fatebreaker:5698:/
+713.2 "Bow Shock" sync /:Fatebreaker:5699:/
+725.7 "Burnished Glory" sync /:Fatebreaker:56A4:/
+
+728.2 "--center--" sync /:Fatebreaker:5908:/ window 30,30
+735.0 "Burnished Glory?"
+737.2 "--Cycle Of Faith--" sync /:Fatebreaker:568A:/ jump 900.0
```

Comment:
```suggestion
737.2 "Cycle Of Faith" sync /:Fatebreaker:568A:/ jump 900.0
```

Reply:
Whoops! Copypasta leftovers from rearranging the order of Cycle blocks!